### PR TITLE
Refactor Functions

### PR DESCRIPTION
**Pull request by segara2410**
- Save action names in `ActionManager` class .
- Add function to play action by name.
- Add function to stop action by clearing the current action pointer.

### CODE REVIEW

File: src/akushon/action_manager.cpp [hunk 1]
```diff
@@ -123,11 +124,36 @@ void ActionManager::set_current_action(const uint8_t & action_id, const Pose & p
...
 
+bool ActionManager::set_current_action(const std::string & action_name)
+{
+  return set_current_action(action_name, *robot_pose);
+}
+
+bool ActionManager::set_current_action(const std::string & action_name, const Pose & robot_pose)
+{
+  for (uint id = 0; id < action_names.size(); id++) {
+    if (action_name == action_names[id]) {
```
Comment (threeal) [inline src/akushon/action_manager.cpp:135]: Use `std::find` instead, see https://en.cppreference.com/w/cpp/algorithm/find.

**Review by threeal — CHANGES_REQUESTED** [review]
See the comment for the requested changes. Also, i think it's better to process action using each action class instead of managing all into one action manager class.

**Review by threeal — APPROVED** [review]
Still not perfect, but it's okay

**Review by maroqijalil — APPROVED** [review]
LGTm

**Pull request merged**